drivers/sensor/vl53l0x/Kconfig

```diff
@@ -32,3 +32,29 @@ config VL53L0X_RECONFIGURE_ADDRESS
 	  value from a sensor, it is powered up and its I2C address is reconfigured
 	  from the manufacturer default (0x29) to the address specified in the
 	  device tree.
+
+config VL53L0X_SETUP_MAX_TIME_FOR_RANGING
```

Comment:
If these values could differ from one sensor to another in a multi sensor application, they should be better proposed as device tree properties.